components/window/events.md

```diff
@@ -168,6 +169,62 @@ If you use the `OnClick` event on a built-in action, it will act as a custom act
 * Use the [VisibleChanged](#visiblechanged) and/or the [StateChanged](#statechanged) events to execute the custom logic on the user actions.
 * Or, use two-way binding for the corresponding Window parameter (e.g., `@bind-Visible`, or `@bind-State`) and toggle its variable from the custom `OnClick` handler.
 
+
+## LeftChanged and TopChanged
+
+These two events fire when the user finishes [moving the window]({%slug window-draggable%}). If you set the `Top` and `Left` parameters of the window, you must update their values in these events - either by handling them yourself, or through using two-way binding.
+
+The values will be in pixels, in a `string` format, and may have many decimal places as reported by the browser.
+
+These events will also fire when the user maximizes the window because then its top and left coordinates become `0`. You can capture this event through the [StateChanged](#statechanged) event that will fire afterwards.
```

Comment:
```suggestion
These events will also fire when the user maximizes the window because then its top and left coordinates become `0px`. You can capture this event through the [StateChanged](#statechanged) event that will fire afterwards.
```